facia-press/app/frontpress/PressedCollectionDeduplication.scala

```diff
@@ -51,7 +57,10 @@ object PressedCollectionDeduplication {
       collectionV: PressedCollectionVisibility,
   ): PressedCollectionVisibility = {
     // Essentially deduplicate the backfill of collectionV using header values values from accum's elements curated and backfill
```

Comment:
Could we expand this comment somehow to explain that `getHeaderURLsFromCuratedAndBackfilledAtDepth(accum, 10)` fetches headerUrls from _previous_ containers, and `getHeaderURLsFromTrailsList(collectionV.pressedCollection.curated)` is the curated cards for the current container? This might help the readability a bit?

Reply:
sounds good!